app/views/directives/edit-webhook-triggers.html

```diff
@@ -1,32 +1,59 @@
+        <i class="pficon pficon-help" data-toggle="tooltip" aria-hidden="true" data-original-title="{{typeInfo}}"></i>
+      </a>
+    </span>
+  </h5>
+  <div ng-repeat="trigger in triggers[webhookTypeName]">
+    <div class="trigger-info">
+      <span class="trigger-url">
+        <copy-to-clipboard is-disabled="trigger.disabled" clipboard-text="bcName | webhookURL : trigger.data.type : trigger.data[webhookType].secret : projectName"></copy-to-clipboard>
+      </span>
+      <span class="visible-xs-inline trigger-actions">
+        <a href="" ng-if="!trigger.disabled" class="action-icon" ng-click="trigger.disabled = true; form.$setDirty()" role="button">
+          <span class="pficon pficon-close" aria-hidden="true" title="Remove"></span>
+          <span class="sr-only">Remove</span>
+        </a>
+        <a href="" ng-if="trigger.disabled" class="action-icon" ng-click="trigger.disabled = false" role="button">
+          <span class="fa fa-repeat" aria-hidden="true" title="Undo"></span>
+          <span class="sr-only">Undo</span>
+        </a>
+      </span>
+      <span class="hidden-xs trigger-actions">
+        <a href="" role="button" ng-if="!trigger.disabled" ng-click="trigger.disabled = true; form.$setDirty()">Remove</a>
+        <a href="" role="button" ng-if="trigger.disabled" ng-click="trigger.disabled = false">Undo</a>
+      </span>
+    </div>
+  </div>
+</div>
+<div class="add-webhook" ng-if="addWebhook">
+  <h5>Add webhook</h5>
   <div class="trigger-info">
     <span class="trigger-url">
-      <copy-to-clipboard is-disabled="trigger.disabled" clipboard-text="bcName | webhookURL : trigger.data.type : trigger.data[(type === 'GitHub') ? 'github' : 'generic'].secret : projectName"></copy-to-clipboard>
+      <ui-select ng-model="triggerMeta.typeToCreate"
+        theme="bootstrap"
+        title="Select a webhooke type"
+        class="select-webhook-type"
+        flex>
+        <ui-select-match placeholder="Select a webhook type">
+          {{ $select.selected }}
+        </ui-select-match>
+        <ui-select-choices repeat="webhookType in triggerMeta.webhookTypes">
+          {{ webhookType }}
+        </ui-select-choices>
+      </ui-select>
     </span>
     <span class="visible-xs-inline trigger-actions">
-      <a href="" ng-if="!trigger.disabled" class="action-icon" ng-click="trigger.disabled = true; form.$setDirty()" role="button">
-        <span class="pficon pficon-close" aria-hidden="true" title="Remove"></span>
-        <span class="sr-only">Remove</span>
-      </a>
-      <a href="" ng-if="trigger.disabled" class="action-icon" ng-click="trigger.disabled = false" role="button">
-        <span class="fa fa-repeat" aria-hidden="true" title="Undo"></span>
-        <span class="sr-only">Undo</span>
+      <a href="" class="action-icon" ng-click="addWebhookTrigger(triggerMeta.typeToCreate)" role="button">
+        <span class="fa fa-plus" aria-hidden="true" title="Add"></span>
```

Comment:
Not really sure what you mean... dont we want to have the same styling here as we have in the for displaying the WH ? Meaning for non-mobile we would have `Add` link and for mobile an icon, as displayed in https://github.com/openshift/origin-web-console/pull/1539#issuecomment-301728175